networkx/generators/community.py

```diff
@@ -464,3 +464,47 @@ def ring_of_cliques(num_cliques, clique_size):
         G.add_edge(i*clique_size+1, (i+1)*clique_size %
                    (num_cliques*clique_size))
     return G
+
+def windmill_graph(n, k):
+    """Generate a windmill graph.
+    A windmill graph is a graph of `n` cliques each of size `k` that are all
+    joined at one node.
+    It can be thought of as taking a disjoint union of `n` cliques of size `k`,
+    selecting one point from each, and contracting all of the selected points.
+    Alternatively, one could generate `n` cliques of size `k-1` and one node
+    that is connected to all other nodes in the graph.
+    Parameters
+    ----------
+    n : int
+      Number of cliques
+    k : int
+      Size of cliques
+    Returns
+    -------
+    G : NetworkX Graph
+      windmill graph with n cliques of size k
+    Raises
+    ------
+    NetworkXError
+        If the number of cliques is less than two
+        If the size of the cliques are less than two
+    Examples
+    --------
+    >>> G = nx.windmill_graph(4, 5)
+    Notes
+    -----
```

Comment:
The doctest fails in travis because there is no blank line between the `Examples` and `Notes` section and the docstring assumes that the output of `G = nx.windmill_graph(4, 5)` should be the rest of the docstring.